### DISCUSSION TITLE
MaintenanceModeHelperInterface Error after Update from 11.0.9 to 11.1.0

### DISCUSSION THREAD

**Discussion by Fabian-Hofer-Mastertent** [pimcore/pimcore#16158] [Q&A / Support] [answered]
Hello Community,

I updated Pimcore from 11.0.9 to 11.1.0 (pimcore/admin-ui-classic-bundle to 1.2 at the same time) but get this error after the composer update:

`Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException {#350
  #message: "You have requested a non-existent service "Pimcore\Tool\MaintenanceModeHelperInterface"."
  #code: 0
  #file: "./vendor/symfony/dependency-injection/Container.php"
  #line: 264
  -id: "Pimcore\Tool\MaintenanceModeHelperInterface"
  -sourceId: null
  -alternatives: []
  trace: {
    ./vendor/symfony/dependency-injection/Container.php:264 { …}
    ./vendor/symfony/dependency-injection/Container.php:212 { …}
    ./vendor/pimcore/pimcore/lib/Console/Application.php:68 { …}
    ./bin/console:38 {
      {closure}
      ›
      ›     return new Application($kernel);
      › };
    }
    ./vendor/autoload_runtime.php:24 { …}
    ./bin/console:20 { …}
  }
}`

I don't understand what the problem could be in this situation because the files and classes seem to be at the right place.

there was a update on the MaintenanceModeHelper as i saw at #15954 in version 11.1.0. Migth there be an issue with this update?

**Comment by dvesh3** [comment, marked as answer]
Please see https://github.com/pimcore/pimcore/issues/16161
